walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
Where is the getRandomQuoteUsingArrowFunctions() method defined?  I don't see it in this PR, is it in another one?

Reply:
It's defined in script.js located in the same directory. The function was already written, and the tutorial just asked us to change the onclick element to call said function. The directions I'm referring to are on page 5 of the Step 2: Fetch tutorial.

Reply:
Ah okay, thanks!